.changeset/afraid-phones-drum.md

```diff
@@ -0,0 +1,8 @@
+---
+"@mds-core/mds-geography": major
+"@mds-core/mds-geography-author": major
+"@mds-core/mds-policy": major
+"@mds-core/mds-policy-author": major
```

Comment:
I think we should probably avoid doing major semver bumps for packages at the moment, see https://github.com/lacuna-tech/mds-core/pull/735#discussion_r709715648